src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
+@import "utils/variables";
+@import "utils/reset";
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
would be better to center it using flex box to body

Reply:
> would be better to center it using flex box to body

why can't I center this element vertically with `align-items: center;`?

body {
  display: flex;
  justify-content: center;
  align-items: center;
  margin: 0;
}